webapp/src/test/java/com/box/l10n/mojito/metrics/HierarchicalNameMapperWithCustomNamingConventionTest.java

```diff
@@ -0,0 +1,322 @@
+package com.box.l10n.mojito.metrics;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.util.HierarchicalNameMapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HierarchicalNameMapperWithCustomNamingConventionTest {
+
+    HierarchicalNameMapper       mapper = HierarchicalNameMapper.DEFAULT;
+    StatsdNamingConventionConfig config;
+    NamingConvention             namingConvention;
+    Meter.Id taggedMeter, untaggedMeter;
+    Tags tags, noTags;
+
+    @BeforeEach
+    void setUp() {
+        config = new StatsdNamingConventionConfig();
+
+        tags = Tags.of("tagA", "valA").and("tagB", "valB");
+        noTags = Tags.empty();
+    }
+
+    @Test
+    void withNamePrefixAndKeyPrefixAndIdentityDoesntChangesNames() {
```

Comment:
I think `@Test` must be placed on with function that are "public"

Reply:
I'm using JUnit 5 here, so there's no need to declare it as public.